video.go

```diff
@@ -136,6 +141,40 @@ func MakeWebM(opts VideoOptions) *exec.Cmd {
 	)
 }
 
+// MakeWebP takes a list of images (as frames) and converts them to a WebP.
+func MakeWebP(opts VideoOptions) *exec.Cmd {
+	if opts.Output.WebP == "" {
+		return nil
+	}
+
+	fmt.Println("Creating WebP...")
+
+	//nolint:gosec
+	return exec.Command(
+		"ffmpeg", "-y",
+		"-r", fmt.Sprint(opts.Framerate),
+		"-i", filepath.Join(opts.Input, textFrameFormat),
+		"-r", fmt.Sprint(opts.Framerate),
+		"-i", filepath.Join(opts.Input, cursorFrameFormat),
+		"-filter_complex",
+		fmt.Sprintf(`[0][1]overlay,scale=%d:%d:force_original_aspect_ratio=1,fps=%d,setpts=PTS/%f,pad=%d:%d:(ow-iw)/2:(oh-ih)/2:%s,fillborders=left=%d:right=%d:top=%d:bottom=%d:mode=fixed:color=%s`,
+			opts.Width-2*opts.Padding, opts.Height-2*opts.Padding,
+			opts.Framerate, opts.PlaybackSpeed,
+			opts.Width, opts.Height,
+			opts.BackgroundColor,
+			opts.Padding, opts.Padding, opts.Padding, opts.Padding,
+			opts.BackgroundColor,
+		),
+		"-vcodec", "libwebp",
+		"-pix_fmt", "yuv420p",
```

Comment:
4:2:0 chroma subsampling can be bad for sharp, high-contrast edges such as fonts.
There's a lossless version of webp, maybe that supports 4:4:4 chroma?